scripts/validate

```diff
@@ -3,27 +3,36 @@ set -e
 
 cd $(dirname $0)/..
 
-GO=${GO-go}
+echo Running: go mod tidy
+go mod tidy
+
+echo Running: go mod vendor
+go mod vendor
```

Comment:
Is this just to get the version flags at https://github.com/k3s-io/k3s/blob/2e91913f545e0f6985af21a68cce7f0d4201cbb7/scripts/build#L24-L37 to work right? If we drop the vendor prefix off those vars, can we stop vendoring at build time?

Reply:
the build script at one point was building shim-runc and hyperkube etc and it looks like you dropped that requirement so we can probably toss the `go mod vendor` and ill give it a shot